server/tracker.go

```diff
@@ -19,17 +20,23 @@ const (
 	// maxBanScore is the score the connection much reach to
 	// be banned by IP.
 	maxBanScore = 3
+
+	// firstPoolNum is the starting number for pools
+	firstPoolNum = 1
+
+	// firstPlayerNum is the starting number for players in a pool
+	firstPlayerNum = uint32(1)
 )
 
 // Tracker is used to track connections to the server.
 type Tracker struct {
-	bannedIPs               map[string]*banData
-	connections             map[net.Conn]*trackerData
+	banDatas                map[string]*banData
```

Comment:
Hmm... Datas isn't a word. The singular for Data is Datum... this one reads weird for me.

Reply:
Me too. Although "data" can technically be a plural word, it is used almost entirely as an uncountable word like water or sand. It's hard to attach a plural meaning to it without sounding odd. I figured the consistency with the other trackers would be more valuable than the grammar strangeness.

Any suggestions? I will think about it too.

Reply:
I think we can just call it banData on the tracker... I don't think its that confusing.

Reply:
Roger that. c16cee5 should accomplish it.